interop_binaries/build.rs

```diff
@@ -47,107 +48,129 @@ fn main() {
                 // any other image metadata (such as exposed ports, the entrypoint, etc), so we can't easily
                 // use it.
                 let client_image_id = {
+                    let image_id_file_name = format!("{out_dir}/janus_interop_client_image_id.txt");
+
                     let client_build_output = Command::new("docker")
                         .args([
+                            "buildx",
                             "build",
-                            "--quiet",
                             "--file=Dockerfile.interop",
                             "--build-arg=PROFILE=small",
                             "--build-arg=BINARY=janus_interop_client",
-                            ".",
+                            "--load",
                         ])
+                        .arg(format!("--iidfile={image_id_file_name}"))
```

Comment:
nit: instead of mixing `args` and `arg` calls, use a single `args` call. (the `format!` parameters can be mixed in by writing `&format!(...)`, since everything will need to be a `&str`.)